src/Workspaces.cc

```diff
@@ -538,11 +538,92 @@ Workspaces::raise(PWinObj* wo)
     if (it == _wobjs.end()) { // no Frame to raise.
         return;
     }
+    if (handleFullscreenBeforeRaise(wo)) {
+        it = find(_wobjs.begin(), _wobjs.end(), wo);
+    }
     _wobjs.erase(it);
 
     insert(wo, true); // reposition and restack
 }
 
+/**
+ * Handles fullscreen windows when raising
```

Comment:
Guess we could make this more descriptive referring to the description in lowerFullscreenWindows.